src/components/Banner/Banner.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box, Flex } from "@chakra-ui/react"
+
+export interface BannerProps {
+  children: string
+}
+
+// TODO: Please get rid of this and use the design system instead!!
+const styles = {
+  banner: { color: "white", bg: "#2B5FCE" },
+  item: {
+    display: "flex",
```

Comment:
i think this is unneeded as chakra's [flex](https://chakra-ui.com/docs/layout/flex) already has it by default

Reply:
Addressed in 8debc69